sycl/include/sycl/ext/intel/experimental/esimd/memory.hpp

```diff
@@ -2873,7 +2873,7 @@ ESIMD_INLINE SYCL_ESIMD_FUNCTION __ESIMD_NS::simd<T, N> lsc_load_2d(
   constexpr uint32_t exDesc = 0x0;
   constexpr uint32_t desc =
       base_desc | cache_mask | transformMask | transposeMask;
-  constexpr uint8_t execSize = 0x0;
+  constexpr uint8_t execSize = N;
```

Comment:
It may work by chance/good-luck, but this doesn't seem correct.
Talked to @vmustya - Victor says the value here should be 1 because lsc_block_load is treated as scalar operation.

Reply:
Fixed, thanks!